README.md

```diff
@@ -1,7 +1,115 @@
+
+Some of the key features and benefits of the KAAP operator include:
+
+- **Easy Deployment**: Deploying an Apache Pulsar cluster on Kubernetes is simplified through declarative configurations and automation provided by the operator.
+
+- **Scalability**: The operator enables effortless scaling of Pulsar clusters by automatically handling the creation and configuration of new Pulsar brokers and bookies as per defined rules.
+
+- **High Availability**: The operator implements best practices for high availability, ensuring that Pulsar clusters are fault-tolerant and can sustain failures without service disruptions.
+
+- **Monitoring and Metrics**: Integrated monitoring and metrics collection allow users to gain insights into the health and performance of Pulsar clusters, making it easier to detect and resolve issues.
+
+- **Lifecycle Management**: The operator takes care of common Pulsar cluster lifecycle tasks, such as cluster creation, upgrade, configuration updates, and graceful shutdowns.
+
+- **Integration with Kubernetes Ecosystem**: The Apache Pulsar Kubernetes Operator seamlessly integrates with other Kubernetes-native tools and features, such as Kubernetes RBAC, Prometheus, and Grafana, enabling a cohesive and comprehensive ecosystem for managing Pulsar clusters.
+
+Whether you are a developer looking to leverage the power of Apache Pulsar in your Kubernetes environment or an operator seeking to streamline the management of Pulsar clusters, the KAAP Operator provides a robust and user-friendly solution. In this documentation, we will explore the installation, configuration, and usage of the operator, empowering you to harness the full potential of Apache Pulsar within your Kubernetes deployments.
+
+## Documentation
+
+Full documentation is available in the [DataStax Streaming Documentation](https://docs.datastax.com/en/streaming/kaap-operator/index.html) or at [this repo's GitHub Pages site](https://datastax.github.io/kaap/docs/).
+
+## Getting started
+
+1. Install the DataStax Helm repository:
+```
+helm repo add kaap https://datastax.github.io/kaap
+helm repo update
+```
+2. Install the Pulsar operator Helm chart:
+```
+helm install kaap kaap/kaap
+```
+Result:
+```
+NAME: kaap
+LAST DEPLOYED: Wed Jun 28 11:37:45 2023
+NAMESPACE: pulsar-cluster
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+```
+3. Ensure pulsar-operator is up and running:
+```
+kubectl get deployment
+```
+Result:
+```
+NAME              READY   UP-TO-DATE   AVAILABLE   AGE
+kaap   1/1     1            1           13m
+```
+
+4. You've now installed KAAP.
+By default, when KAAP is installed, the PulsarCluster CRDs are also created.
+This setting is defined in the Pulsar operator values.yaml file as `crd: create: true`.
+
+5. To see available CRDs:
+```
+kubectl get crds | grep datastax
+```
+Result:
+```
+autorecoveries.pulsar.oss.datastax.com           2023-05-12T16:35:59Z
```

Comment:
all of them are now with kaap instead of pulsar

autorecoveries.kaap.oss.datastax.com